docs/team/hanjiyao.md

```diff
@@ -12,24 +12,29 @@ ContacX will allow for efficient management of a large list of contacts, allowin
 Given below are my contributions to the project.
 
 * **New Feature**: Bulk Operations
-  * What it does: `To be added soon`
-  * Justification: `To be added soon`
+  * What it does: allow user to create multiple commands together by range batch or chain
```

Comment:
```suggestion
  * What it does: Allow user to create multiple commands together by range batch or chain
```
I would suggest starting each point with a capital letter for readability and consistency. This applies to the subsequent points in this document

Reply:
noted, will change accordingly